src/databricks/labs/ucx/cli.py

```diff
@@ -81,7 +81,8 @@ def skip(w: WorkspaceClient, schema: str | None = None, table: str | None = None
         return None
     ctx = WorkspaceContext(w)
     if table:
-        return ctx.table_mapping.skip_table(schema, table)
+        is_view = ctx.tables_crawler.is_view(schema, table)
+        return ctx.table_mapping.skip_table_or_view(schema, table, is_view)
```

Comment:
```suggestion
        return ctx.table_mapping.skip_table_or_view(schema, table)
```
is_view should be a private method, otherwise it's a detrimental change for the future modifications of this code and would make it more confusing down the line. please refactor

Reply:
done